test/index.ts

```diff
@@ -57,6 +63,92 @@ export function runTests(
                 runBenchmarksForCanisters(canisterNamesArray));
         });
     }
+
+    // TODO we should work on what the UI looks like if there are multiple of these things running
+    if (shouldFuzz && canisterNames !== undefined) {
+        // TODO right now we just test the first canister...that might be good enough for now?
```

Comment:
Are these TODOs related to and outdated with the one above?

Reply:
All removed and refactored